cmd/test/fixtures/db-query-log-exporter-custom.json

```diff
@@ -12,7 +12,7 @@
           "log_min_error_statement": "ERROR",
           "log_line_prefix": "%m :%r :%u @ %d :[%p] : %a :"
         },
-        "exporter_id": "9e740000-331b-4dec-89b0-4e59b81e9019"
+        "exporter_id": "129f7c97-81ae-47c7-8f9e-40ab4390093f"
```

Comment:
**Note for reviewer**: I am re-using some response.json files from `integrations_test.go`. So I have changed the exporter id in log exporter json's to same as in `list-telemetry-providers.json` to do some assertions on exporter id.